### PR TITLE
Update download script, cleaning script, environment.yaml and final report .rmd file

### PR DESCRIPTION
**Pull request by SiqiTao**

### CODE REVIEW

File: results/score_results.csv [hunk 1]
```diff
@@ -1,5 +1,10 @@
 score_type,Logistic Regression,Logistic Regression Optimized
+<<<<<<< HEAD
```
Comment (danfke) [inline results/score_results.csv:2]: Merge conflict didnt get solved - shouldnt be a problem though

**Review by danfke — APPROVED** [review]
Looks good, thanks doris

**Review by gn385x — APPROVED** [review]
We discussed the change in group meeting. Approved!

**Comment by danfke** [comment]
altair_saver==0.5.0
imbalanced-learn==0.8.1

File: environment.yaml [hunk 2]
```diff
@@ -11,5 +11,8 @@ dependencies:
   - pip=21.3.1=pyhd8ed1ab_0
   - python=3.10.0
   - scikit-learn==1.0.1
+  - pandoc==2.16.2
+  - altair_saver==0.5.0
```
Comment (danfke) [inline environment.yaml:13]: delete these lines

Comment (SiqiTao) [reply environment.yaml:13]: done

**Review by iamMoid — APPROVED** [review]

**Pull request merged**